dev-docs/bidders/visiblemeasures.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: bidder
+title: VisibleMeasures
+description: Prebid VisibleMeasures Bidder Adapter
+biddercode: visiblemeasures
+usp_supported: true
+gdpr_supported: true
+coppa_supported: true
+schain_supported: true
+floors_supported: true
+media_types: banner, video, native
+pbjs: true
+pbs: true
+pbs_app_supported: true
+sidebarType: 1
+---
+
+### Bid Params
+
+{: .table .table-bordered .table-striped }
+| Name          | Scope    | Description           | Example   | Type      |
+|---------------|----------|-----------------------|-----------|-----------|
+| `placementId`      | optional | Placement Id         | `'0'`    | `'string'` |
+| `endpointId`      | optional | Endpoint Id         | `'0'`    | `'string'` |
+
+### Note
+
+For the prebid server and prebid.js you only need to use one parameter: either placementId or endpointId
```

Comment:
If it doesn't matter, I would recommend you specify one as `required` and the others as optional. This is a little confusing for users of your adapter :wink: 

Also an example is always nice. Maybe you have a test `placementId` that a dev can tinker around with.

Reply:
Sorry, but in our case we need one of two necessarily, but it doesn't matter which one